test/app/App/Panel/AddToken/index.test.js

```diff
@@ -82,23 +82,107 @@ describe('setting token address', () => {
     expect(contractAddressInput.textContent).toBe('')
   })
 
-  it('should update add token navigation when an address is entered', async () => {
+  it('should update add token navigation when an invalid contract address is entered', async () => {
```

Comment:
a more descriptive test name can be helpful when scanning tests to figure out how this component should work. for example `it('should navigate to a token error screen when ...')`

Reply:
sounds good